config.ru

```diff
@@ -9,6 +9,10 @@ log.level = Logger::INFO unless opts['debug']
 
 opts['log'] = log
 
+# prepare statsd
+require 'statsd-ruby'
+$STATSD = Statsd.new(:host => opts['statsd_host'], :port => opts['statsd_port'])
```

Comment:
This should just be a constant like `STATSD`, global variables should only be used when the value will vary, and probably shouldn't be used.